### PR TITLE
Update mac mini page

### PR DESCRIPTION
**Pull request by benhyland**
Give more details about how to get access to the mac mini.

Give urls for all the pages it currently displays.

### CODE REVIEW

File: docs/team/dashboard_mac_mini.md [hunk 1]
```diff
@@ -8,6 +8,33 @@ following:
...
+## Outstanding pull requests
+
+These are shown by [Fourth Wall](https://github.com/alphagov/fourth-wall).
+
+The URL for our dashboard is
+```
+https://alphagov.github.io/fourth-wall/?token=<your github access token>&team=alphagov/team-government-paas-readonly&github.gds_token=<your github.gds access token>&github.gds_team=government-paas/read-only
+```
+
+Replace the placeholders with read-only access tokens from github and github.gds.
```
Comment (timmow) [inline docs/team/dashboard_mac_mini.md]: Is it worth [linking here](https://github.com/blog/1509-personal-api-tokens) to give some context on how to generate these?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 site/
+env/
+
```
Comment (timmow) [inline .gitignore]: Can we remove the blank newline? `</nitpick>`

File: docs/team/dashboard_mac_mini.md [hunk 3]
```diff
@@ -28,4 +55,5 @@ on the Dashboard. It can be found
 
 We use the [Pingdom public status page](http://stats.pingdom.com/ejtodj13fqqx) ([documented here](https://help.pingdom.com/hc/en-us/articles/205386171-Public-Status-Page)) to display the uptime and current status of our healthcheck application. The Pingdom account is shared with other Government-as-a-Platform services, such as Notify, and there is only one shared dashboard for the entire account.
 
-We use the [Super Auto Refresh](https://chrome.google.com/webstore/detail/super-auto-refresh/kkhjakkgopekjlempoplnjclgedabddk?hl=en) Chrome extension to refresh the page.
\ No newline at end of file
+We use the [Super Auto Refresh](https://chrome.google.com/webstore/detail/super-auto-refresh/kkhjakkgopekjlempoplnjclgedabddk?hl=en) Chrome extension to refresh the page.
+
```
Comment (timmow) [inline docs/team/dashboard_mac_mini.md]: Can we remove the blank newline? </nitpick>

**Comment by benhyland** [comment]
Addressed comments.

**Pull request merged**